test/scenarios/fork/ynETHWithdrawals.t.sol

```diff
@@ -0,0 +1,544 @@
+
+    bool public isHolesky;
+
+    address public constant user = address(0x12345678);
+    address public constant receiver = address(0x987654321);
+
+    uint256 public nodeId = 2;
+    uint256 public withdrawalAmount = 32 ether;
+
+    uint256 tokenId;
+    uint256 secondsToFinalization;
+    uint256 amountOut;
+
+    StakingNode public stakingNode;
+
+    // ------------------------------------------
+    // Setup
+    // ------------------------------------------
+
+    function setUp() public override {
+        super.setUp();
+
+        isHolesky = block.chainid == 17000;
+
+        if(isHolesky) {
+            stakingNode = StakingNode(payable(address(stakingNodesManager.nodes(nodeId))));
+
+            vm.deal(user, 10_000 ether);
+
+            setupForVerifyWithdrawalCredentials(nodeId, "test/data/holesky_wc_proof_1916455.json");
+
+            secondsToFinalization = 1 days;
+            vm.prank(actors.ops.WITHDRAWAL_MANAGER);
+            ynETHWithdrawalQueueManager.setSecondsToFinalization(secondsToFinalization);
+        }
+    }
+
+    // ------------------------------------------
+    // Withdrawal flow tests
+    // ------------------------------------------
+
+    function testVerifyWithdrawalCredentials() public {
+        if (!isHolesky) return;
+
+        ValidatorProofs memory _validatorProofs = getWithdrawalCredentialParams();
+
+        vm.expectRevert(bytes4(keccak256("NotStakingNodesOperator()")));
+        stakingNode.verifyWithdrawalCredentials(
+            uint64(block.timestamp),
+            _validatorProofs.stateRootProof,
+            _validatorProofs.validatorIndices,
+            _validatorProofs.withdrawalCredentialProofs,
+            _validatorProofs.validatorFields
+        );
+
+        uint256 _unverifiedStakedETH = stakingNode.unverifiedStakedETH();
+
+        vm.prank(actors.ops.STAKING_NODES_OPERATOR);
+        stakingNode.verifyWithdrawalCredentials(
+            uint64(block.timestamp),
+            _validatorProofs.stateRootProof,
+            _validatorProofs.validatorIndices,
+            _validatorProofs.withdrawalCredentialProofs,
+            _validatorProofs.validatorFields
+        );
+
+        assertEq(stakingNode.unverifiedStakedETH(), _unverifiedStakedETH, "testVerifyWithdrawalCredentials: E0"); // validator already verified withdrawal credentials
+    }
+
+    function testQueueWithdrawal() public {
+        if (!isHolesky) return;
+
+        // uint256 _amount = withdrawalAmount;
+        // fuzzing here will fail on too many inputs rejected
+        // vm.assume(_amount > 1e9); // _amount > 1 gwei
+        // vm.assume(_amount % 1e9 == 0); // _amount must be a whole Gwei amount
+        // vm.assume(_amount <= uint256(IEigenPodManager(stakingNodesManager.eigenPodManager()).podOwnerShares(address(stakingNode))));
+
+        testVerifyWithdrawalCredentials();
+
+        vm.expectRevert(bytes4(keccak256("NotStakingNodesOperator()")));
```

Comment:
separate test case

Reply:
fixed in https://github.com/yieldnest/yieldnest-protocol/pull/124/commits/e40cb6f14cb1feb306eafa7a9ba859b03944625e.